.dependabot/config.yml

```diff
@@ -0,0 +1,13 @@
+version: 1
+update_configs:
+    - package_manager: "java:gradle"
+      directory: "./library/"
```

Comment:
I believe we'll need 3 entries total in this file, with this entry being for `maps-ktx` and a new one added for `maps-utils-ktx`

Reply:
Copy/pasted from AMU 😓  Updating